src/components/Rockets.js

```diff
@@ -1,8 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets, addReservation, deleteReservation } from '../redux/rockets/rocketsSlice';
+import '../styles/Rockets.css';
 
 function Rockets() {
+  const dispatch = useDispatch();
+  const { rockets } = useSelector((store) => store.rockets);
+
+  const handleAdd = (id) => {
+    dispatch(addReservation(id));
+  };
+
+  const handleDelete = (id) => {
+    dispatch(deleteReservation(id));
+  };
+
+  useEffect(() => {
```

Comment:
- [ ] `Rockets` is the default view, so its data would be fetched when the application starts, kindly make sure that the rockets view gets rendered when the app is opened the first time